### PR TITLE
Add more VerifyResult states

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo/issues/2538

### CODE REVIEW

File: src/neo/Ledger/VerifyResult.cs [hunk 1]
```diff
@@ -47,6 +47,26 @@ public enum VerifyResult : byte
         /// </summary>
         PolicyFail,
 
+        /// <summary>
+        /// Indicates that the <see cref="Transaction"/> it's too big.
+        /// </summary>
+        TooBig,
```
Comment (Liaojinghui) [inline src/neo/Ledger/VerifyResult.cs:53]: I would suggest using `TransactionOverSize` or something similar to replace `TooBig`

File: src/neo/Network/P2P/Payloads/Transaction.cs [hunk 2]
```diff
@@ -371,7 +371,7 @@ public virtual VerifyResult VerifyStateDependent(ProtocolSettings settings, Data
                 else
                 {
                     if (!this.VerifyWitness(settings, snapshot, hashes[i], witnesses[i], net_fee, out long fee))
-                        return VerifyResult.Invalid;
+                        return VerifyResult.InvalidSignature;
```
Comment (erikzhang) [inline src/neo/Network/P2P/Payloads/Transaction.cs:374]: This is not necessarily an invalid signature, right?

Comment (shargon) [reply src/neo/Network/P2P/Payloads/Transaction.cs:374]: We can use `Invalid` if you prefer

Comment (shargon) [reply src/neo/Network/P2P/Payloads/Transaction.cs]: Fixed

**Review by Qiao-Jin — DISMISSED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**